### DISCUSSION TITLE
AutoGen Team, calling Functions Problem pls Help

### DISCUSSION THREAD

**Discussion by lfdy-suicide** [microsoft/autogen#2354] [Q&A]
Hello guys, i have a goal to create automated autogen team with Writer, Validator, Legal aspekt, so Writer write and ask for a suggestions about text, as i want to create Legal Agent based on a book info i have a lot of struggles, i dont know at all how to configure that, the best result i reached is creating a separate function that a working perfecntly, but when i paste it inside my Autogen main script this output for me summarization of Writers output instead of providing suggested data. If you dont understand what i mean, for example i Ask to write about History of Racing, and Writer mentioned Global Rome Race 2008(idk xD) and in the book that Legal Agent have there is additional more deep information about this race, so i want Legal agent to provide a additional suggested data, but instead i receive in my lurge script just summarization of text when its calling. 
I attach to you my book, and 2 codes, one with separate script where its working(im not crazy) and second one with my large script
[Legal.txt](https://github.com/microsoft/autogen/files/14947192/Legal.txt)
[pls help.zip](https://github.com/microsoft/autogen/files/14947265/pls.help.zip)

**Comment by ekzhu** [comment]
Can you post the code and the text directly without using a zip file?

**Reply by lfdy-suicide** [reply]
My legal.txt file is a huge book(80 pages) so i dont want to paste there 80 pages directly.
Im neewbie there (in git also) , if you know how to do this please explain to me how to paste it correctly 🙂

**Reply by lfdy-suicide** [reply]
also i might refference to this page https://docs.llamaindex.ai/en/stable/getting_started/starter_example/ , i use this code and my txt file to it
